diff --git a/node_modules/eslint-config-expo/flat/default.js b/node_modules/eslint-config-expo/flat/default.js
index ce828bf..5821266 100644
--- a/node_modules/eslint-config-expo/flat/default.js
+++ b/node_modules/eslint-config-expo/flat/default.js
@@ -3,10 +3,10 @@ const expoConfig = require('./utils/expo.js');
 const reactConfig = require('./utils/react.js');
 const typescriptConfig = require('./utils/typescript.js');
 const { allExtensions } = require('./utils/extensions.js');
-const { defineConfig } = require('eslint/config');
+// const { defineConfig } = require('eslint/config');
 const globals = require('globals');
 
-module.exports = defineConfig([
+module.exports = [
   ...coreConfig,
   ...typescriptConfig,
   ...reactConfig,
@@ -43,4 +43,4 @@ module.exports = defineConfig([
   {
     files: ['*.web.*'],
   },
-]);
+];
